Maquetado/index.html

```diff
@@ -10,10 +10,93 @@
 </head>
 
 <body>
+  <div class="front-end">
```

Comment:
Use a more generic classname for this class to add reusability

Reply:
this is not updated with the page-title branch.